Codo-Baggins/covid_comparisons
# Feature/comparison card styles

## What's this PR do?
- Add styles to `ComparisonCards` and `ComparisonDetails`
- Represents death as icons where each icon = 50 deaths
- Closes responsiveness issue #12 

## Where should the reviewer start?
- `ComparisonCards.tsx` and `ComparisonDetails.tsx`

## How should this be manually tested?
- Pick a comparison and see the styles on the next page
- Change between the comparisons to see the numbers update

## Screenshots/GIFs
- <img width="443" alt="Screen Shot 2021-01-12 at 12 05 08 PM" src="https://user-images.githubusercontent.com/58453566/104360702-b9ac3900-54ce-11eb-8d85-8437bcd02f24.png">
